twitter_pr.rs

```diff
@@ -0,0 +1,96 @@
+extern crate reqwest;
+
+use std::sync::mpsc::{Receiver, channel};
```

Comment:
I think you may need the async versions of Receiver and channel instead: https://docs.rs/tokio/0.3.0/tokio/sync/mpsc/index.html.

Reason: the std::sync versions will block, but async functions are supposed to avoid blocking whenever possible. Happy to go into more detail on this, it's a complicated subject and I don't know how much you already know!